solutions/swb-app/src/projects/projectPlugin.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  CreateProjectRequest,
+  DeleteProjectRequest,
+  GetProjectRequest,
+  ListProjectsRequest,
+  Project,
+  UpdateProjectRequest
+} from '@aws/workbench-core-accounts';
+import { GetProjectsRequest } from '@aws/workbench-core-accounts/lib/models/projects/getProjectsRequest';
+import { AuthenticatedUser } from '@aws/workbench-core-authorization';
+import { PaginatedResponse } from '@aws/workbench-core-base';
```

Comment:
Is `swb-app` still supposed to avoid dependencies on other projects in the repo?

Reply:
if not, `GetProjectsRequest` should probably be exposed in `@aws/workbench-core-accounts` index file